#! /bin/sh /usr/share/dpatch/dpatch-run
## fix-trunks-sqlite3.dpatch by Diego Iastrubni <diego.iastrubni@xorcom.com>
##
## DP: Trunks page was not working when the sql engine was set to sqlite3, the reason is a malformed sql query

@DPATCH@
diff -urNad upstream~/amp_conf/htdocs/admin/modules/core/functions.inc.php upstream/amp_conf/htdocs/admin/modules/core/functions.inc.php
--- upstream~/amp_conf/htdocs/admin/modules/core/functions.inc.php	2007-06-08 03:42:50.000000000 +0300
+++ upstream/amp_conf/htdocs/admin/modules/core/functions.inc.php	2007-07-26 15:28:42.000000000 +0300
@@ -1543,7 +1543,7 @@
 		// TODO: sqlite work arround - diego 
 		// need to reorder the trunks in PHP code
 		// remember, in sqlite3 we need to use escape
-		$unique_trunks = sql("SELECT * FROM globals WHERE variable LIKE 'OUT_\\%' escape '\\' ORDER BY variable","getAll"); 
+                $unique_trunks = sql("SELECT * FROM globals WHERE variable LIKE 'OUT\\_%' escape '\\' ORDER BY variable","getAll");
 	}
 	else
 	{
@@ -1896,6 +1896,8 @@
 function core_routing_setroutepriority($routepriority, $reporoutedirection, $reporoutekey)
 {
 	global $db;
+	global $amp_conf;
+
 	$counter=-1;
 	foreach ($routepriority as $tresult) 
 	{
